component-server-parent/component-server/src/main/java/org/talend/sdk/component/server/service/qualifier/ComponentServer.java

```diff
@@ -24,8 +24,7 @@
 
 import javax.inject.Qualifier;
 
-@Qualifier
-@Retention(RUNTIME)
+@Qualifier @Retention(RUNTIME)
```

Comment:
it's a bit strange, maybe we need to change java_formatter eclipse file if it did this